includes/sanitizers/class-amp-blacklist-sanitizer.php

```diff
@@ -39,7 +39,14 @@ private function strip_attributes_recursive( $node, $bad_attributes, $bad_protoc
 
 				// on* attributes (like onclick) are a special case
 				if ( 0 === stripos( $attribute_name, 'on' ) ) {
```

Comment:
It would be better to change the check to skip when we explicitly match an attribute_name of `on` since attributes like `onclick` are not allowed:

> Attribute names starting with on (such as onclick or onmouseover) are disallowed in AMP HTML. The attribute with the literal name on (no suffix) is allowed.

via https://www.ampproject.org/docs/reference/spec.html